src/main/scala/ai/privado/javascript/processor/JavascriptProcessor.scala

```diff
@@ -50,6 +76,20 @@ object JavascriptProcessor {
             logger.debug(
               s"Total Sinks identified : ${cpg.tag.where(_.nameExact(Constants.catLevelOne).valueExact(CatLevelOne.SINKS.name)).call.tag.nameExact(Constants.id).value.toSet}"
             )
+            val codelist = cpg.call
+              .where(
+                _.methodFullName
+                  .filterNot(_.matches(".*operator.*"))
+              )
+              .map(item => (item.methodFullName, item.location.filename))
+              .dedup
+              .l
+            println(s"size of code : ${codelist.size}")
+            codelist.foreach(println)
+            println("Above we printed methodFullName")
```

Comment:
Check if we need this or not in logs.

Reply:
@khemrajrathore convert this to debug logs instead of println